1.8/usage/tutorials/tutorial/loadbalancing.md

```diff
@@ -0,0 +1,40 @@
+---
+post_title: Load-balancing
+nav_title: Load-balancing
+menu_order: 7
+---
+
+# Prerequisites
+* A [running DC/OS cluster](/docs/1.8/tutorial/cli/) with [the DC/OS CLI installed](link to installation instructions].
+* [app2 and marathon-lb](/docs/1.8/tutorial/app2/) deployed and running in your cluster.
+
+# Objective
+In the last session, we will scale our application to multiple instances, and learn how internal and external services chose which instance to use once our application has been scaled.
+
+# Steps
+Load-balancers decide which instance of an app internal or external services should use. With DC/OS you have two different built-in load-balancer options: : [Marathon-LB](https://dcos.io/docs/1.8/usage/service-discovery/marathon-lb/) and [Named virtual IPs](https://dcos.io/docs/1.8/usage/service-discovery/load-balancing-vips/).
+We have seen both before in the context of service discovery and especially marathon-lb when making app2 publically available.
+Now we will explore both options in more detail.
+* Therefore we first scale app2 to two instances: `dcos marathon app update /dcos-101/app2 instances=2`
+* Marathon-lb
+    * Check the app2 as before via *http://public-node>10000*. When you do this repeatedly you should see the request being served by different instances of app2.
+    * You can also check the marathon-lb stats via `http://<public-node>:9090/haproxy?stats`
+* Named VIPs
+    * We have seen named vips before for service discovery inside our cluster
+    * SSH to the leading master node: `dcos node ssh --master-proxy --leader`
+    * `curl dcos-101app2.marathon.l4lb.thisdcos.directory:10000`. When you do this repeatedly you should see the request being served by different instances.
+* Scale app2 back to one instance: `dcos marathon app update /dcos-101/app2 instances=1`
+
+
+
+# Outcome
+We used two different methods to load-balance requests two different instances of our app.
+
+# Deep Dive
+With two options, we actually need to decide on one of them.
```

Comment:
Really like your suggested sentence (many others as well, just wanted to it out here ;-) )